### PR TITLE
inline emojis

### PR DESCRIPTION
**Pull request by mattn**
See https://github.com/damus-io/damus/issues/2003

image

### CODE REVIEW

**Comment by fiatjaf** [comment]
What is the id of that note?

**Comment by mattn** [comment]
https://njump.me/nevent1qvzqqqqqqypzq2aj4wlutzft00dg77x4x6pdjy7vnfzxk30pry5lpy6a3l0uks9aqy88wumn8ghj7mn0wvhxcmmv9uq32amnwvaz7tmjv4kxz7fwv3sk6atn9e5k7tcqyp7a3z7nlr4p4eqhyckmpkcql6rses4p23f5rlpcpaptg8upe4a07c50gha

**Comment by fiatjaf** [comment]
Now I regret having merged this PR since the current way is so glorious:
2024-02-13-112158_881x952_scrot

**Comment by cxplay** [comment]
So, how does it look in Telegram InstantView?

**Comment by mattn** [comment]
still not deployed?

**Comment by alltheseas** [comment]
> Now I regret having merged this PR since the current way is so glorious

The proper term is neck paca maximalism

**Comment by fiatjaf** [comment]
Still glorious @cxplay:

photo_2024-02-13_12-35-58
photo_2024-02-13_12-36-00

**Comment by koteitan** [comment]
How do we have name space in kind:30030?

`:import kubipaca: :L: :R: :U: :D: :N:`

and so on(same).

**Comment by cxplay** [comment]
> Still glorious @cxplay:
> ...
damn

**Comment by mattn** [comment]
https://yabu.me/nevent1qvzqqqqqqypzq2aj4wlutzft00dg77x4x6pdjy7vnfzxk30pry5lpy6a3l0uks9aqy88wumn8ghj7mn0wvhxcmmv9uq32amnwvaz7tmjv4kxz7fwv3sk6atn9e5k7tcqyp7a3z7nlr4p4eqhyckmpkcql6rses4p23f5rlpcpaptg8upe4a07c50gha

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "devDependencies": {
     "prettier": "^3.0.3",
     "prettier-plugin-tailwindcss": "^0.5.6",
-    "tailwindcss": "^3.3.5"
+    "tailwindcss": "^3.4.1"
```
Comment (dtonon) [inline package.json:5]: @mattn the Tailwind update is necessary for the emoji rendering?

**Pull request merged**